pkg/storage/unified/resource/index_server_distributor.go

```diff
@@ -73,8 +73,8 @@ func (c *RingClient) RemoteAddress() string {
 	return c.Conn.Target()
 }
 
-const RingKey = "unified-storage-ring"
-const RingName = "unified_storage_ring"
+const RingKey = "index-server-ring"
+const RingName = "index_server_ring"
```

Comment:
nit: our opportunity to make the casing of the constant values more consistent.

```suggestion
const RingName = "index-server-ring"
```

Reply:
We use `RingName` as a metrics prefix, where we should use underscores.